FreeRTOS/FreeRTOS
# Extend qemu virt riscv demo

Description
-----------
Build on Katsuhiro Suzuki's original RISC-V Virt QEMU demo to include all the common demo tasks and an Eclipse project that builds the updated makefile.

Test Steps
-----------
Ensure the Eclipse project builds, as that will also ensure the makefile builds.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
